lib/yard/handlers/c/handler_methods.rb

```diff
@@ -67,7 +67,9 @@ def handle_method(scope, var_name, name, func_name, _source_file = nil)
             register_visibility(obj, visibility)
             find_method_body(obj, func_name)
             obj.explicit = true
-            obj.add_tag(Tags::Tag.new(:return, '', 'Boolean')) if name =~ /\?$/
+            if name =~ /\?$/ && obj.tags(:return).empty?
```

Comment:
This implementation should probably be normalized to handle overloads as per the Ruby handler:

https://github.com/lsegal/yard/blob/master/lib/yard/handlers/ruby/method_handler.rb#L41-L49

Reply:
Is there any existing patterns to sharing logic between Ruby and C handlers?
Or should I just copy+paste?